elixir/multistage.exs

```diff
@@ -0,0 +1,30 @@
+Mix.install([
+  {:dagger, github: "wingyplus/dagger", sparse: "sdk/elixir", ref: "iss-5333"},
```

Comment:
I will change it after https://github.com/dagger/dagger/pull/5526 gets merged.

Reply:
The dagger/dagger#5526 already merged. But we still need to release the library to the hex.pm.